src/rcms/fm/app/level1/HCALMasker.java

```diff
@@ -196,9 +197,11 @@ protected void setMaskedFMs() {
 
     VectorT<StringT> maskedFMsVector = new VectorT<StringT>();
     for (QualifiedResource qr : level2list) {
-      if (qr.getName().equals(EvmTrigFM)) { 
-         qr.getResource().setRole("EvmTrig");
-         functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("EVM_TRIG_FM", new StringT(qr.getName())));
+      if (qr.getName().equals(EvmTrigFM)) {
+        if (functionManager.RunType.equals("local")){
+          qr.getResource().setRole("EvmTrig");
+          functionManager.getHCALparameterSet().put(new FunctionManagerParameter<StringT>("EVM_TRIG_FM", new StringT(qr.getName())));
+         }
```

Comment:
Better set `EVM_TRIG_FM` to "" for global here for clarity.
This is needed for auto-Masking because the config will contain the EvmTrig trio, so `pickEvmTrig()` will pick an FM, but the EvmTrig apps will be masked later in `setMaskedFMs`.